package.json

```diff
@@ -27,10 +27,17 @@
     "url": "https://github.com/strongloop/loopback-soap/issues"
   },
   "devDependencies": {
+    "chai": "^3.2.0",
     "eslint": "^3.12.2",
-    "eslint-config-loopback": "^7.0.0"
+    "eslint-config-loopback": "^8.0.0",
+    "loopback": "^3.0.0",
```

Comment:
@rmg I am hesitant to remove it for now.. I will do end to end test (not just module level test) and see if it works using 'lb soap' command and REST APIs can be invoked from APIExplorer. If this works then, I will remove "loopback": "^3.0.0", in another PR.. I have another work item to add more test cases to this module in next sprint.